control/control.openSUSE.xml

```diff
@@ -505,19 +505,21 @@ textdomain="control"
             </proposal>
 
             <volumes config:type="list">
-                <!-- / volume: 5 GiB - 20 GiB, *2 if snapshots are used -->
+                <!-- / volume: 16 GiB - 30 GiB, like Kubic -->
                 <volume>
                     <mount_point>/</mount_point>
                     <fs_type>btrfs</fs_type>
 
-                    <desired_size config:type="disksize">10 GiB</desired_size>
-                    <min_size config:type="disksize">5 GiB</min_size>
-                    <max_size config:type="disksize">20 GiB</max_size>
-                    <weight config:type="integer">60</weight>
+                    <desired_size config:type="disksize">25 GiB</desired_size>
+                    <min_size config:type="disksize">16 GiB</min_size>
+                    <max_size config:type="disksize">30 GiB</max_size>
+                    <weight config:type="integer">80</weight>
 
                     <snapshots config:type="boolean">true</snapshots>
-                    <snapshots_configurable config:type="boolean">true</snapshots_configurable>
-                    <snapshots_percentage config:type="integer">100</snapshots_percentage>
+                    <snapshots_configurable config:type="boolean">false</snapshots_configurable>
+
+                    <!-- You don't want to miss the / volume -->
```

Comment:
I do not get this comment, can you be more specific what is problem?

Reply:
For the filesystem on a transactional rootfs you really need to make sure the root filesystem is configured exactly as proposed. The comment is actually copied from the same lines of config from skelcd-control-CAASP, as this unification applies to all 3 systems using transactional hosts (CaaSP, Kubic, and now here)

Reply:
ok, so at least it is not clear everywhere :)